test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -242,3 +243,15 @@
     (is (thrown-with-msg? IllegalStateException
                           #"Found an old and unuspported database migration.*"
                           (migrate! *db*)))))
+
+(deftest test-upgrade-migration
```

Comment:
seems to me like the code comment is wrong, and that this is actually testing a 3.x upgrade